nginx/tests/integration/charms/k8s/pyproject.toml

```
@@ -0,0 +1,12 @@
+[project]
+name = "test"
+version = "0.1.0"
+description = "Test charm for integration tests"
+requires-python = ">=3.8"
+dependencies = [
+    "ops==2.*",
```

```suggestion
    "ops==3.*",
```
Need to change this too when dropping 3.8 -- plus of course `nginx/pyproject.toml`.